packaging/rhel/syslog-ng.spec

```diff
@@ -1,8 +1,8 @@
 %global ivykis_ver 0.36.1
 
 Name: syslog-ng
-Version: 3.15.1
-Release: 2%{?dist}
+Version: 3.16.1
+Release: 1%{?dist}
```

Comment:
@czanik: Can you review this change, please?
As we see we need to modify the `Release` part of the spec file always: `Release` is a package revision number, therefore we have to reset this number on every release.
See [rpm doc](http://ftp.rpm.org/max-rpm/s1-rpm-build-creating-spec-file.html).

Reply:
LGTM

I changed the spec file in a way, that you only need to update the version number here, but not in the other parts of the file.

Reply:
Thanks!
I remember the previous change to this file, when the exact version number in filenames changed to a glob pattern.
https://github.com/balabit/syslog-ng/blob/b5ec5ffd4d204038319bd8521e6f4a7441fade18/packaging/rhel/syslog-ng.spec#L333-L337